tbtadm/controller.cpp

```diff
@@ -415,26 +442,38 @@ void tbtadm::Controller::printTree(
     for (const auto& device : map)
     {
         auto last = device.first == map.rbegin()->first;
-        m_out << indentation << "|\n";
-        m_out << indentation << "+- " << device.second.m_desc[0];
+        m_out << indentation << symbols[SYMBOL_PIPE] << std::endl;
+        if (last)
+            m_out << indentation << symbols[SYMBOL_L] << device.second.m_desc[0];
+        else
+            m_out << indentation << symbols[SYMBOL_PLUS] << device.second.m_desc[0];
```

Comment:
Maybe replace the `if` with `m_out << indentation << symbols[last ? SYMBOL_L : SYMBOL_PLUS] << device.second.m_desc[0];`?
A matter of taste.